DigitalLearningSolutions.Data/Models/User/AdminUser.cs

```diff
@@ -41,6 +41,8 @@ public class AdminUser : User
         public bool IsCmsAdministrator => ImportOnly && IsContentManager;
         public bool IsCmsManager => IsContentManager && !ImportOnly;
 
+        public bool Active { get; set; }

```

Comment:
Active was removed from the GetAdminUserById query in the AdminDataService, but this property still remains on the model. Because it does not get populated, the bool will default to false, and therefore does not accurately represent the Active status of the AdminUser. Thus it cannot be relied upon in the code logic. 

As we didn't have this before, this is not a major issue (I highlight in other comments where it may be an issue), but we will need some consideration as to what to do with it. I think it should probably be left as it is, and the GetAdminUserById query should return Active. Out of curiosity, why was Active removed from the GetAdminUserById query?

Reply:
Active field is added in the query so this kept as it is